debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -219,7 +218,7 @@ protected void initReplicationSlot() throws SQLException, InterruptedException {
                 this.createReplicationSlot();
             }
 
-            AtomicLong xlogStart = new AtomicLong();
+            final AtomicReference<Lsn> xlogStart = new AtomicReference<>();
```

Comment:
Would it be possible to use `queryAndMap` here in order to avoid the `AtomicReference` usage?

Reply:
Done, thanks!